modules/systemd.nix

```
@@ -29,6 +29,23 @@ let
       in "${key}=${value'}";
   };
 
+  importServicesFromPkg = pkg:
+    let
+      isService = (path: type: (builtins.match ".service$" path) == null);
```

This currently only includes files that aren't matching (`== null`) 😅 

I would do it like this (and also include `.target` files):


```suggestion
      isService = (path: type: lib.hasSuffix ".service" path || lib.hasSuffix ".target" path);
```